tests/openebs-pool-container-failure_test.go

```diff
@@ -166,4 +199,67 @@ var _ = Describe("BDD of openebs pool container failure experiment", func() {
 		})
 	})
 
+	//Matching the Resource Verison after Chaos
+	Context("Check Resource Version of pool container", func() {
+
+		It("Should check for the change in Resource Version after Chaos", func() {
+			resourceVersionSumAfter := 0
+			csp_rv, err := client.CoreV1().Pods("openebs").List(metav1.ListOptions{LabelSelector: "app=cstor-pool"})
+			Expect(err).To(BeNil(), "fail to get the csp pods")
+			for _, podSpec := range csp_rv.Items {
+				i2, _ := strconv.Atoi(v.ResourceVersion)
+				resourceVersionSumAfter = resourceVersionSumAfter + i2
+			}
+
+			Expect(resourceVersionSumBefore-resourceVersionSumAfter).NotTo(Equal(0), "The Resource Version does not change")
+			fmt.Printf("The Resource Version changes")
+
+		})
+	})
+
+	//Matching the ContainerIDs after Chaos
+	Context("Check ContainerIDs after chaos", func() {
+
+		It("Should check for the change in ContainerIDs of csp pod", func() {
+			var container_id2 [9]string
+			l := 0
+			c := 0
```

Comment:
better name like `containerIDChanged` of boolean type and use false/ true instead of 0/1